docs/ref/utils.txt

```diff
@@ -947,20 +947,34 @@ appropriate entities.
     Returns ``True`` if ``value`` is naive, ``False`` if it is aware. This
     function assumes that ``value`` is a :class:`~datetime.datetime`.
 
-.. function:: make_aware(value, timezone=None)
+.. function:: make_aware(value, timezone=None, is_dst=None)
 
     Returns an aware :class:`~datetime.datetime` that represents the same
     point in time as ``value`` in ``timezone``, ``value`` being a naive
     :class:`~datetime.datetime`. If ``timezone`` is set to ``None``, it
     defaults to the :ref:`current time zone <default-current-time-zone>`.
 
-    This function can raise an exception if ``value`` doesn't exist or is
-    ambiguous because of DST transitions.
+    When ``pytz`` is available the exception ``pytz.AmbiguousTimeError``
+    will be raised if you try to make ``value`` aware
+    during a DST transition where the same time occurs twice (when reverting
+    from DST). Setting ``is_dst`` to ``True`` or ``False`` will avoid the exception
+    by choosing if the time is pre-transition or post-transition respectively.
+    ``is_dst`` has no effect when ``pytz`` is not installed.
+
+    When ``pytz`` is available the exception ``pytz.NonExistentTimeError``
+    will be raised if you try to make ``value`` aware during a DST transition
+    such that the time never occurred (when entering into DST). You can catch
```

Comment:
Oh that's cool then, and solves one of the issues I had where I thought I wasn't really doing anything with regards to NonExtistentTimeError. I'll write some tests and correct the docs. Thanks for pointing that out.